cookbooks/bach_spark/templates/default/spark-env.sh.erb

```diff
@@ -0,0 +1,3 @@
+export HADOOP_CONF_DIR="/etc/hadoop/conf"
+export HDP22_BASE_DIR="/usr/hdp/current/hadoop-client"
```

Comment:
The dependency on the hadoop basedir has been frustrating me. Perhaps follow [Using Spark's "Hadoop Free" Build](http://spark.apache.org/docs/latest/hadoop-provided.html) instructions? (Particularly, `hadoop classpath`.)